example/operator/20-garden.yaml

```
@@ -216,6 +216,7 @@ spec:
     #   featureGates:
     #     SomeGardenerFeature: true
     #   goAwayChance: 0.001
+    #   adminKubeconfigMaxExpiration: 24h
```

Wouldn't this naming make more sense? The flag is also called this way.

```suggestion
    #   shootAdminKubeconfigMaxExpiration: 24h
```